### PR TITLE
R18 Roadmap

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- Merge dev to master, get R18 roadmap public

### CODE REVIEW

**Review by joebochill — APPROVED** [review]

File: src/data/doc-it/R18Roadmap.ts [hunk 1]
```diff
@@ -0,0 +1,509 @@
...
+                author: 'PX Blue',
+            },
+            {
+                name: 'Dropdown Toolbar',
+                description: 'Add component for PX Blue Dropdown Toolbar',
+                year: 2020,
+                quarter: 'Q4',
+                status: 'backlog',
+                author: 'PX Blue',
+            },
```
Comment (huayunh) [inline src/data/doc-it/R18Roadmap.ts:87]: This is finished

File: src/data/doc-it/R18Roadmap.ts [hunk 2]
```diff
@@ -0,0 +1,509 @@
...
+export const roadmap: RoadmapBucket[] = [
+    {
+        name: 'Figma Stickersheet Components',
+        description: 'Updates to the components in the designer stickersheet',
+        type: 'design',
+        items: [
+            {
+                name: 'Additional Material Components',
+                description:
+                    'Add components for Chips, Bottom Sheet, Cards, Sliders, Snackbar, and Tooltip',
```
Comment (huayunh) [inline src/data/doc-it/R18Roadmap.ts:34]: Also added: Switches, Avatars, FABs

**Review by huayunh — CHANGES_REQUESTED** [review]

**Comment by huayunh** [comment]
release notes associated with the changes above is updated in https://github.com/pxblue/doc-it/pull/275

File: src/data/doc-it/R18Roadmap.ts (hunk 1 above)
Comment (joebochill) [reply src/data/doc-it/R18Roadmap.ts:87]: we can move it into Q3 and mark it done

**Review by huayunh — APPROVED** [review]

**Pull request merged**